metrotranscom/doorway
# feat: add low listings volume info cards

# Pull Request Template

## Issue Overview

This PR addresses #116 


## Description

- Add 3 info cards at the end of the listings list with additional information.
- Style the info cards according to the Figma mocks.
- Restyle primary and secondary buttons to have the correct colors for default state, hover state and click state according to Figma mocks. Updated storybook to be able to see the changes clearly.
- Change the Bloom primary color from blue-700 to blue-900 according to the Figma colors for the buttons and change other colors accordingly. Checked with UX on whether this should be done or if we just need to adjust the color for the buttons, haven't heard back yet. This affects the individual listing page and makes the background color for some sections too dark. Happy to roll back/discuss this part further.
- Add new text and translations (from Google Translate) for the 3 cards mentioned. I adjusted the Spanish translations a bit to have the same verb tenses across all the new ones.

What's missing and will be in future PRs
- Logic to only include these cards in the last page once we have pagination on the front end. Will wait for that to happen first.
- Links for the 3 buttons.
- Zero results page which has these same 3 cards + a section saying that there are 0 results.

Listings page:
![image](https://github.com/metrotranscom/doorway/assets/67125998/2f67a135-7069-4113-8893-7335d2e0c507)

Individual listing page:
This changed indirectly because of the color changes. The blue in the background is primary-light which is now too dark.
![image](https://github.com/metrotranscom/doorway/assets/67125998/42835399-0909-4898-a709-2e5278afe6fa)


## How Can This Be Tested/Reviewed?

- Open the public site
- Go to the listings page
- Scroll to the bottom of the listings page to see the info cards. Compare the styling to the mocks.
- Hover over and click the buttons. Compare the styling to the mocks.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
